src/pymovements/utils/parsing.py

```diff
@@ -150,9 +150,8 @@ def parse_eyelink(
 
     for line in lines:
         for pattern_dict in compiled_patterns:
-            match = pattern_dict['pattern'].match(line)
 
-            if match:
+            if match := pattern_dict['pattern'].match(line):
```

Comment:
I think the walrus operator is handy for matching regexes (I always hated it to write two lines for this).

So keep this please.